src/rules/select-style.ts

```diff
@@ -51,7 +51,7 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
   defaultOptions: [{ mode: METHOD }],
   create: (context, [{ mode }]) => {
     return {
```

Comment:
To avoid reading store multiple times, we could read this only on initialization instead:

```suggestion
    const storeName = readNgRxStoreNameFromSettings(context.settings)

    return {
```